src/shogun/classifier/PluginEstimate.cpp

```diff
@@ -1,33 +1,37 @@
 /*
  * This software is distributed under BSD 3-clause license (see LICENSE file).
  *
- * Authors: Soeren Sonnenburg, Sergey Lisitsyn, Heiko Strathmann, 
+ * Authors: Soeren Sonnenburg, Sergey Lisitsyn, Heiko Strathmann,
  *          Evan Shelhamer
  */
 
-#include <shogun/lib/common.h>
-#include <shogun/io/SGIO.h>
+#include <shogun/classifier/PluginEstimate.h>
+#include <shogun/distributions/LinearHMM.h>
 #include <shogun/features/StringFeatures.h>
-#include <shogun/labels/Labels.h>
+#include <shogun/io/SGIO.h>
 #include <shogun/labels/BinaryLabels.h>
+#include <shogun/labels/Labels.h>
 #include <shogun/labels/RegressionLabels.h>
-#include <shogun/distributions/LinearHMM.h>
-#include <shogun/classifier/PluginEstimate.h>
+#include <shogun/lib/common.h>
 
 using namespace shogun;
 
 CPluginEstimate::CPluginEstimate(float64_t pos_pseudo, float64_t neg_pseudo)
-: CMachine(), m_pos_pseudo(1e-10), m_neg_pseudo(1e-10),
-	pos_model(NULL), neg_model(NULL), features(NULL)
+    : CMachine(), m_pos_pseudo(1e-10), m_neg_pseudo(1e-10), pos_model(NULL),
+      neg_model(NULL), features(NULL)
 {
 	SG_ADD(
-	    &m_pos_pseudo, "pos_pseudo", "pseudo count for positive class");
+	    &m_pos_pseudo, "pos_pseudo", "pseudo count for positive class",
+	    ParameterProperties::MODEL);
 	SG_ADD(
-	    &m_neg_pseudo, "neg_pseudo", "pseudo count for negative class");
+	    &m_neg_pseudo, "neg_pseudo", "pseudo count for negative class",
+	    ParameterProperties::MODEL);
 	SG_ADD(
-	    &pos_model, "pos_model", "LinearHMM modelling positive class.");
+	    &pos_model, "pos_model", "LinearHMM modelling positive class.",
+	    ParameterProperties::MODEL);
 	SG_ADD(
-	    &neg_model, "neg_model", "LinearHMM modelling negative class.");
+	    &neg_model, "neg_model", "LinearHMM modelling negative class.",
+	    ParameterProperties::MODEL);
```

Comment:
assuming these should all be MODEL?

Reply:
I think this class doesn’t work anyways. But yes model